docs/dns/domains.rst

```diff
@@ -115,6 +116,25 @@ Field details:
     write operations that did not trigger publication, such as rewriting an
     RRset with identical values.
 
+``zonefile``
+    :Access mode: write-only
+
+    Optionally, includes a string in zonefile format with zone data to be
```

Comment:
suggestion: `with record data`. (We try to avoid use of the word `zone` elsewhere, but not strictly; I think we should do a better job at this. KISS for users.)

Reply:
difficult, but done a bit